deepchecks/tabular/dataset.py

```diff
@@ -1061,6 +1065,172 @@ def datasets_share_date(cls, *datasets: 'Dataset') -> bool:
 
         return True
 
+    @property
+    def _dataset_description(self) -> t.Tuple[
+        t.List[t.Union[str, int]],
+        pd.DataFrame,
+        pd.DataFrame
+    ]:
+        data = self.data
+        datetime_name = self.datetime_name
+        index_name = self.index_name
+        label_name = self.label_name
+        features = self.features
+        categorical_features = self.cat_features
+        numerical_features = self.numerical_features
+
+        columns = features[::]
```

Comment:
why columns = features? why not just the dataframe columns?

Reply:
what if provided dataframe contains much more columns than there are features that are interesting for the user

![image](https://user-images.githubusercontent.com/71635444/171128622-bb6393e6-ec64-464c-8e62-3a1fc5a4046d.png)

Reply:
If it's a topic of "we're showing at most 7 columns", I agree that we should first show the label, index, datetime, features, and only last the other columns. 
But if the dataframe has a total of 5 columns (e.g. 1 label, 3 features, 1 other) then we should show the other.

Reply:
Ok so as the expected behavior we will do the following:
First display Index, Datetime, Label, then features, and then if there is some space for more columns we will display other columns which are not features,